ballerina/tests/47_unary_timestamp_service.bal

```diff
@@ -50,4 +50,26 @@ service "UnaryTimestampService" on ep47 {
             check caller->complete();
         }
     }
+
+    remote function serverStreamTime(TimestampServiceTimestampCaller caller, time:Utc value) returns error? {
+        time:Utc responseTime = checkpanic time:utcFromString("2008-12-03T11:15:30.120Z");
```

Comment:
Let's replace `checkpanic` with `check`.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/372/commits/69f160eca3668e350a01ca297ba3404aae9ce7d2